src/api/templates/emails/domain_added_notice.html

```diff
@@ -0,0 +1,251 @@
+        <table
+          align="center"
+          border="0"
+          cellpadding="0"
+          cellspacing="0"
+          role="presentation"
+          style="width: 100%"
+        >
+          <tbody>
+            <tr>
+              <td
+                style="
+                  direction: ltr;
+                  font-size: 0px;
+                  padding: 20px 0;
+                  text-align: center;
+                "
+              >
+                <!--[if mso | IE]><table role="presentation" border="0" cellpadding="0" cellspacing="0"><tr><td class="" style="vertical-align:top;width:600px;" ><![endif]-->
+                <div
+                  class="mj-column-per-100 mj-outlook-group-fix"
+                  style="
+                    font-size: 0px;
+                    text-align: left;
+                    direction: ltr;
+                    display: inline-block;
+                    vertical-align: top;
+                    width: 100%;
+                  "
+                >
+                  <table
+                    border="0"
+                    cellpadding="0"
+                    cellspacing="0"
+                    role="presentation"
+                    style="vertical-align: top"
+                    width="100%"
+                  >
+                    <tbody>
+                      <tr>
+                        <td
+                          align="left"
+                          style="
+                            font-size: 0px;
+                            padding: 10px 25px;
+                            word-break: break-word;
+                          "
+                        >
+                          <div
+                            style="
+                              font-family: helvetica;
+                              font-size: 20px;
+                              line-height: 1;
+                              text-align: left;
+                              color: #1b2352;
+                            "
+                          >
+                            Dear {{ first_name }} {{ last_name }},
+                          </div>
+                        </td>
+                      </tr>
+                      <tr>
+                        <td
+                          align="left"
+                          style="
+                            font-size: 0px;
+                            padding: 10px 25px;
+                            word-break: break-word;
+                          "
+                        >
+                          <div
+                            style="
+                              font-family: helvetica;
+                              font-size: 20px;
+                              line-height: 1;
+                              text-align: left;
+                              color: #1b2352;
+                            "
+                          >
+                            A new domain has been added to our systems. To
```

Comment:
Please change the first sentence to "A new sending domain has been added."

Reply:
@itsmostafa changed as requested with ea67d89694fc5d1b924e713049be3bcef582aced